site/en/posts/site-refactor-2022.md

```diff
@@ -0,0 +1,92 @@
+
+Because of this, we decided the complexity of maintaining our configuration separately from our site (Static Site Scaffold) was no longer worthwhile, so we only kept individual pieces we had seen real need to re-use across projects. Our first set of work, then, was [migrating our Eleventy config into our codebase](https://github.com/chromeos/chromeos.dev/issues/147), and modularizing our [localization logic](https://github.com/chromeos/chromeos.dev/issues/145) and [markdown config](https://github.com/chromeos/chromeos.dev/issues/146) into separate modules. The localization logic was especially useful to modularize, as, right before our initial deploy, there was a bug in Node 12 that broke our ability to use [full-icu](https://www.npmjs.com/package/full-icu) and left us pinned on a specific Node 12 version. During the the modularization, we learned that the bug had been fixed and that, in later versions of Node, it was no longer needed at all, further simplifying our codebase.
+
+## A Shift in Tooling
+
+We launched ChromeOS.dev August of 2020. Between when we launched the site and the end of the year, a tectonic shift in development tooling happened, starting with [Snowpack 2.0](https://www.npmjs.com/package/snowpack) and [esbuild](https://www.npmjs.com/package/esbuild), then [WMR](https://www.npmjs.com/package/wmr), and finally [Vite 2.0](https://www.npmjs.com/package/vite). The bundler ecosystem embraced ES modules and a "batteries included mentality, making them lightning fast while removing the need to configure complex features, like code splitting and hot module reloading. The result? Faster development that required less maintenance and produced more optimized code. We're in.
+
+Our original codebase held Eleventy, Gulp, Rollup, and Browser Sync together with string and duct tape. It was slow. Configuration was spread out throughout many files and systems. It was fragile. After some experimenting, we decided that Vite was the best fit for us. It had a great ecosystem, it made choices that aligned with our sensibilities, it was the fastest of the three, and it worked really well for our needs (not building a full JS app, compiling Sass separate from our JS pipeline, working with both workers and service workers). It took about a month to get all the pieces in place, but we finally managed to [migrate our build and local serving to Vite](https://github.com/chromeos/chromeos.dev/issues/150). A number of interesting _new_ modules came as a result of migrating to Vite.
```

Comment:
nit: The parenthetical makes this feel like a run on. Maybe something like "...and it worked really well for our needs. Those needs being not building a full JS app..."